pkgs/kernels/r36/default.nix

```diff
@@ -0,0 +1,133 @@
+{ applyPatches
+, lib
+, fetchFromGitHub
+, l4t-xusb-firmware
+, realtime ? false
+, kernelPatches ? [ ]
+, structuredExtraConfig ? { }
+, argsOverride ? { }
+, buildLinux
+, gitRepos
+, ...
+}@args:
+
+let
+  useOe4tKernelSrc = false;
+
+  oe4tKernelSrc = fetchFromGitHub {
+    owner = "OE4T";
+    repo = "linux-tegra-5.10";
+  };
+in
+buildLinux (args // {
```

Comment:
As a reminder, attribute set merging through (`//`) is not deep -- be careful if there's anything you need to keep from `args`! For example, `extraMeta.branch = "5.15"` will wipe out any existing `extraMeta` from `args` -- likewise, any `extraMeta` from `argsOverride` would replace that!